src/actions/guides/database/validating-saving-deleting.cr

```diff
@@ -8,96 +8,96 @@ class Guides::Database::ValidatingSavingDeleting < GuideAction
 
   def markdown
     <<-MD
-    ## Setting up a form
+    ## Setting up an operation
 
-    When you set up a model, a `{ModelName}::BaseForm` will be created so that you can inherit
+    When you set up a model, a `{ModelName}::SaveOperation` will be created so that you can inherit
     from it and customize validations, callbacks, and what fields are allowed to be
-    filled. `{ModelName}::BaseForm` automatically defines a form field for each model field.
+    filled. `{ModelName}::SaveOperation` automatically defines a form field for each model field.
 
     We’ll be using the migration and model from the [Querying
     guide](#{Guides::Database::Querying.path}). Once you have that set up, let’s set
-    up a form:
+    up a save operation:
 
     ```crystal
-    # src/forms/user_form.cr
-    class UserForm < User::BaseForm
+    # src/operations/save_user.cr
+    class SaveUser < User::SaveOperation
     end
     ```
 
-    > You can have more than one form object to save a record in different ways.
-    For example, default Lucky apps have a `SignUpForm` specifically for handling
+    > You can have more than one operation object to save a record in different ways.
+    For example, default Lucky apps have a `SignUserUp` specifically for handling
     user sign ups.
 
     ## Allowing params to be saved
 
     By default you won’t be able to set any data from params. This is a security
     measure to make sure that parameters can only be set that you want to allow
     users to fill out. For example, you might not want your users to be able to
-    set an admin status through the `UserForm`.
+    set an admin status through the `SaveUser` operation.
 
-    To allow a field to be saved to the database, use the `fillable` macro:
+    To allow a field to be saved to the database, use the `permit_columns` macro:
```

Comment:
We shuold clarify that this is for saving a  param to the database.  If you set it directly like `SaveUser.create!(name: "foo")` or set it in the object with `name.value = "foo")` it will save.

So maybe `To allow users to set columns from JSON/form params,  use the `permit_columns` macro: